### PR TITLE
FPLA-2798: Send Respondent Standard Notifications

### PR DESCRIPTION
**Pull request by samuelzerozero**
Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/FPLA-2798

### Change description ###
Send Respondent Standard Notifications

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/GeneratedOrderEventHandler.java [hunk 1]
```diff
@@ -61,7 +63,7 @@ public void sendOrderByPost(final GeneratedOrderEvent orderEvent) {
 
     private void sendNotificationToEmailServedRepresentatives(final CaseData caseData,
                                                               final DocumentReference orderDocument) {
-        List<Representative> emailRepresentatives = caseData.getRepresentativesByServedPreference(EMAIL);
+        Set<String> emailRepresentatives = representativesInbox.getEmailsByPreference(caseData, EMAIL);
 
         if (!emailRepresentatives.isEmpty()) {
```
Comment (Rob-Doherty) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/GeneratedOrderEventHandler.java:68]: @samuelzerozero should there be a similar set of changes for [SendNoticeOfHearingHandler](https://github.com/hmcts/fpl-ccd-configuration/blob/master/service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/SendNoticeOfHearingHandler.java) to cover the 3rd AC for
- Notice of New Hearing

Comment (Rob-Doherty) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/GeneratedOrderEventHandler.java:68]: ?

Comment (samuelzerozero) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/GeneratedOrderEventHandler.java:68]: `  
@Async
    @EventListener
    public void notifyRepresentatives(final SendNoticeOfHearing event) {
        final CaseData caseData = event.getCaseData();

        SERVING_PREFERENCES.forEach(servingPreference -> {
            NotifyData notifyData =
                newHearingContent.buildNewNoticeOfHearingNotification(
                    caseData, event.getSelectedHearing(), servingPreference);

            representativeNotificationService
                .sendToRepresentativesByServedPreference(
                    servingPreference, NOTICE_OF_NEW_HEARING, notifyData, caseData);
        });
    }
`
representativeNotificationService.sendToRepresentativesByServedPreference has been already modified to gather the extra requirement https://github.com/hmcts/fpl-ccd-configuration/blob/9ebcfdeb0b4c8bc919b9011fc4238793c1407a94/service/src/main/java/uk/gov/hmcts/reform/fpl/service/representative/RepresentativeNotificationService.java#L25

Comment (Rob-Doherty) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/GeneratedOrderEventHandler.java:68]: cheers for explaining @samuelzerozero

**Review by SooryaSu — APPROVED** [review]

**Pull request merged**